# SC-Genomics
This is a repo that contains relevant info regarding single-cell DNA (exome and whole genome) sequencing computational analysis and interpretation, including protocols, pipelines, peak calling algorithms, benchmarking articles, and applications of these techniques.

The specific focus is on single-cell cancer genomics.

## Early Work

### Tumour evolution inferred by single-cell sequencing
Nicholas Navin, et al. Tumour evolution inferred by single-cell sequencing. _Nature_ (2011). https://doi.org/10.1038/nature09807

This was the first demonstration of single-cell DNA sequencing across the whole genome, and was used to reconstruct and model the evolutionary history of the tumor. The authors were able to identify distinct subpopulations that likely gave rise to clonal expansions with divergent structural variations containing potent oncogenes. 

### Genome-wide copy number analysis of single cells (_Nature Protocols_)
Timour Baslan, et al. Genome-wide copy number analysis of single cells. _Nature Protocols_ (2012). https://doi.org/10.1038/nprot.2012.039

This is an accompanying detailed protocol based on the previous paper. 

## Later Papers

Darlan C. Minussi, Michael D. Nicholson, Hanghui Ye, et al. Breast tumours maintain a reservoir of subclonal diversity during expansion. _Nature_ (2021). https://doi.org/10.1038/s41586-021-03357-x

Kaile Wang, Tapsi Kumar, et al. Archival single-cell genomics reveals persistent subclones during DCIS progression. _Cell_ (2023). https://doi.org/10.1016/j.cell.2023.07.024

Yiyun Lin, Junke Wang, et al. Normal breast tissues harbour rare populations of aneuploid epithelial cells. _Nature_ (2024). https://doi.org/10.1038/s41586-024-08129-x

## Good Review Articles on this topic:

Anna S. Nam, Ronan Chaligne & Dan A. Landau. Integrating genetic and non-genetic determinants of cancer evolution by single-cell multi-omics. _Nature Reviews Genetics_ (2021). https://doi.org/10.1038/s41576-020-0265-5

## Other modalities (scATAC-seq, scHi-C, etc.) for cancer genomics

### Single-cell ATAC-seq for cancer genomics and tumor evolution
Derek H. Janssens, et al. Automated CUT&Tag profiling of chromatin heterogeneity in mixed-lineage leukemia. _Nature Genetics_ (2021). https://doi.org/10.1038/s41588-021-00941-9

Steven J. Wu, Scott N. Furlan, et al. Single-cell CUT&Tag analysis of chromatin modifications in differentiation and tumor progression. _Nature Biotechnology_ (2021). https://doi.org/10.1038/s41587-021-00865-z

Kaile Wang, Zhenna Xiao, Yun Yan, et al. Simple oligonucleotide-based multiplexing of single-cell chromatin accessibility. _Molecular Cell_ (2021). https://doi.org/10.1016/j.molcel.2021.09.026

Xiaotao Wang, Yu Luan, & Feng Yue. EagleC: A deep-learning framework for detecting a full range of structural variations from bulk and single-cell contact maps. _Science Advances_ (2022). https://doi.org/10.1126/sciadv.abn9215

Lei Chang, Yang Xie, et al. Droplet Hi-C enables scalable, single-cell profiling of chromatin architecture in heterogeneous tissues. _Nature Biotechnology_ (2024). https://doi.org/10.1038/s41587-024-02447-1

